### PR TITLE
tests: check that we should not find `/dev/disk/by-id/google*` device links on non GCP

### PR DESCRIPTION
**Pull request by HuijingHei**
See https://issues.redhat.com/browse/OCPBUGS-13754 
https://github.com/coreos/fedora-coreos-tracker/issues/1475#issuecomment-1551857322

### CODE REVIEW

File: tests/kola/var-mount/scsi-id/test.sh [hunk 1]
```diff
@@ -39,3 +40,9 @@ if ! grep -q "63-scsi-sg3_symlink.rules" $tempfile; then
 fi
 
 rm -f $tempfile
+
+# https://issues.redhat.com/browse/OCPBUGS-13754
+links=$(find /dev/disk/by-id/ -name google*)
```
Comment (travier) [inline tests/kola/var-mount/scsi-id/test.sh]:
```suggestion
links=$(find /dev/disk/by-id/ -iname "*google*")
```

File: tests/kola/var-mount/scsi-id/test.sh [hunk 2]
```diff
@@ -39,3 +40,9 @@ if ! grep -q "63-scsi-sg3_symlink.rules" $tempfile; then
...
+if [ -n "${links:-}" ]; then
```
Comment (travier) [inline tests/kola/var-mount/scsi-id/test.sh]:
```suggestion
if [[ -n "${links:-}" ]]; then
```

**Review by travier — COMMENTED** [review]
Some suggestions / nits.

**Comment by dustymabe** [comment]
I feel like maybe we should create a new test for this since it feels weird to just add it into another (seemingly unrelated) test.

IIUC this doesn't require a specific disk setup so we could make it a non-exclusive test (that way we won't end up spawning an extra VM but just re-use the non-exclusive-tests VM.

**Comment by HuijingHei** [comment]
> I feel like maybe we should create a new test for this since it feels weird to just add it into another (seemingly unrelated) test.
> 
> IIUC this doesn't require a specific disk setup so we could make it a non-exclusive test (that way we won't end up spawning an extra VM but just re-use the non-exclusive-tests VM.

SGTM. Thanks @dustymabe and @travier for the reivew.

File: tests/kola/disks/device-link [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+## kola:
+##   platforms: qemu
```
Comment (dustymabe) [inline tests/kola/disks/device-link]: add ?

```
##   exclusive: false
```

**Comment by dustymabe** [comment]
maybe name the test `tests/kola/disks/no-google-device-links`

File: tests/kola/disks/no-google-device-links [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+## kola:
+##   platforms: qemu
```
Comment (travier) [inline tests/kola/disks/no-google-device-links]: I'd say that we should run this on all platforms but GCP?

Comment (HuijingHei) [reply tests/kola/disks/no-google-device-links]: Yes, you are correct, maybe I should change to `!gcp` ?

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**